src/confluent_kafka/src/Admin.c

```diff
@@ -4369,6 +4470,52 @@ static PyObject *Admin_c_ListOffsetsResultInfos_to_py (const rd_kafka_ListOffset
         return NULL;
 }
 
+static PyObject *Admin_c_DeletedRecords_to_py (const rd_kafka_topic_partition_list_t *c_topic_partitions) {
+    
+    size_t c_topic_partition_cnt = c_topic_partitions->cnt;
+    PyObject *result = NULL;
+    PyObject *DeleteRecords_type = NULL;
+    size_t i;
+
+    DeleteRecords_type = cfl_PyObject_lookup("confluent_kafka", "DeleteRecords");
+ 
+    if(!DeleteRecords_type){
+        cfl_PyErr_Format(RD_KAFKA_RESP_ERR__INVALID_ARG, "Unable to load DeleteRecords type");
+        goto raise;
+    }
+
+    result = PyDict_New();
+    for(i=0; i<c_topic_partition_cnt; i++){
+        PyObject *value = NULL;
+        rd_kafka_topic_partition_t *c_topic_partition = &c_topic_partitions->elems[i];
+        if (c_topic_partitions->elems[i].err) {
+            value = KafkaError_new_or_None(c_topic_partitions->elems[i].err, rd_kafka_err2str(c_topic_partitions->elems[i].err));
+        } else {
+            PyObject *args = NULL;
+            PyObject *kwargs = NULL;
+            kwargs = PyDict_New();
+            cfl_PyDict_SetLong(kwargs, "low_watermark", c_topic_partitions->elems[i].offset);
+            args = PyTuple_New(0);
+            value = PyObject_Call(DeleteRecords_type, args, kwargs);
+            Py_DECREF(args);
+            Py_DECREF(kwargs);
+            if (value == NULL)
+                goto raise;
+        }
+        PyObject *key = c_part_to_py(c_topic_partition);
+        PyDict_SetItem(result, key, value);
+        Py_DECREF(key);
+        Py_DECREF(value);
+    }
+
+    Py_DECREF(DeleteRecords_type);
+    return result;
+raise:
+    Py_XDECREF(result);
+    Py_XDECREF(DeleteRecords_type);
+    return NULL;
+}
```

Comment:
Use same style as other functions.

Reply:
I was asking about indentations and line breaks.

`static PyObject *Admin_c_DeletedRecords_to_py (const rd_kafka_topic_partition_list_t *c_topic_partitions, size_t c_topic_partition_cnt)` is not expected here. You can revert this count part.